vms/platformvm/camino_vm_test.go

```diff
@@ -701,6 +750,19 @@ func TestAdminProposals(t *testing.T) {
 	require.NoError(err)
 	require.True(applicantAddrState.IsNot(as.AddressStateConsortium))
 
+	// Set kyc admin role
+	addrStateTx, err = vm.txBuilder.NewAddressStateTx(
+		test.FundedKeys[0].Address(),
+		false,
+		as.AddressStateBitRoleKYCAdmin,
+		test.FundedKeys[0].Address(),
+		[]*secp256k1.PrivateKey{test.FundedKeys[0]},
+		nil,
+	)
```

Comment:
```suggestion
	// Set kyc admin role
	addrStateTx, err = vm.txBuilder.NewAddressStateTx(
		kycAdminKey.Address(),
		false,
		as.AddressStateBitRoleKYCAdmin,
		rootAdminKey.Address(),
		[]*secp256k1.PrivateKey{test.FundedKeys[0]},
		nil,
	)
```
Should this also be updated with the new var names?